external_parser/event_processors/joined_event.h

```diff
@@ -324,6 +326,77 @@ struct ccb_joined_event : public typed_joined_event {
   }
 };
 
+struct slates_joined_event : public typed_joined_event {
+  MultiSlotInteraction multi_slot_interaction;
+  float reward;
+  float original_reward;
+
+  ~slates_joined_event() = default;
+
+  bool is_skip_learn() const override { return multi_slot_interaction.skip_learn; }
+  void set_skip_learn(bool sl) override { multi_slot_interaction.skip_learn = sl; }
+
+  void set_apprentice_reward() override {}
+
+  bool fill_in_label(v_array<example *> &examples) const override {
+    size_t slot_index = 0;
+
+    for (auto *ex : examples) {
```

Comment:
Do we also need to fill the weight for every example, same as we do on ccb fill_in_label?